lib/geo/GeoStyle.js

```diff
@@ -436,6 +437,27 @@ class GeoStyle {
           ],
         },
       }),
+      {
+        id: 'locations-midblocks',
+        source: 'locations',
```

Comment:
IMHO we need to switch to a model where each `FcPaneMap` instance has its own `GeoStyle` instance.  Right now, changing the map options blows away the `locations` GeoJSON source data (tested this with both layer toggles and aerial mode) - I _suspect_ this is because we're rebuilding the style from the ground up each time we call `GeoStyle.get`.

Reply:
nvm, found a simpler solution here.